printer_mods/Mckurt/Switchwire Y Belt Tensioner/Readme.md

```diff
@@ -0,0 +1,27 @@
+![parts](Images/asembly.png)
+![parts](Images/mod.png)
+
+This mod for Voron Switchwires Y axis Belt tension, you don't need cut panels
+
+Fasteners
+
+-2x M3x8mm SHCS
```

Comment:
Remove empty lines between list items, and add a space after the `-`, so Markdown makes it an actual list.  If you do something like

```
- 2x M3x8mm SHCS
- 2x M5x10mm SHCS
- 1x M5x10mm BHCS
```

Markdown will make it a list that looks like

- 2x M3x8mm SHCS
- 2x M5x10mm SHCS
- 1x M5x10mm BHCS